### PR TITLE
[RTBF] Get full title and description

### PR DESCRIPTION
**Pull request by Urgau**
### Before submitting a *pull request* make sure you have:
- [X] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [X] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [ ] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [X] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [X] Bug fix
- [X] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Get the full title and description of RTBF by downloading the real webpage and searching og metadata.

### CODE REVIEW

**Comment by dstftw** [comment]
1. Provide concrete examples of what this is supposed to fix and can't be extracted from embed page.
2. Any added code must be fail safe. Read coding conventions and fix.

**Comment by Urgau** [comment]
Exemple with : https://www.rtbf.be/auvio/detail_the-good-doctor-s01?id=2386922

youtube-dl : 
 - Title : The Good Doctor S01
 - Description : C'est le premier jour de Shaun à l'hôpital Saint Bonaventure. Alors qu'il est arrivé en retard de cinq minutes, Claire tente de prendre sa défense, mais il se...

original : 
 - Title :  The Good Doctor S01 02 - Sale boulot
 - Description : C'est le premier jour de Shaun à l'hôpital Saint Bonaventure. Alors qu'il est arrivé en retard de cinq minutes, Claire tente de prendre sa défense, mais il se fait recadrer par Melendez, qui l'affecte aux tâches subalternes pour la journée : corvée de consultations d'urgence et formalités administratives. Shaun ne comprend pas qu'il soit parfois nécessaire de "mentir" aux patients pour ne pas les effrayer : coup sur coup, il informe un patient opéré de la prostate qu'il a de fortes chances d'être impuissant et Stéphanie, une patiente atteinte d'une tumeur, que celle-ci est maligne et risque fort de la tuer...

The embed page provides only a short version of the title and the description they can't be extracted from the embed page.
I read the coding convention but I don't see where my code is not fail safe.

File: youtube_dl/extractor/rtbf.py [hunk 1]
```diff
@@ -69,9 +69,20 @@ class RTBFIE(InfoExtractor):
 
     def _real_extract(self, url):
         live, media_id = re.match(self._VALID_URL, url).groups()
+        
+        webpage = self._download_webpage(url, media_id)
```
Comment (dstftw) [inline youtube_dl/extractor/rtbf.py]: Breaks on failure.

File: youtube_dl/extractor/rtbf.py [hunk 2]
```diff
@@ -69,9 +69,20 @@ class RTBFIE(InfoExtractor):
...
+
+        title = self._og_search_title(webpage)
```
Comment (dstftw) [inline youtube_dl/extractor/rtbf.py]: Breaks if not found.

**Comment by Urgau** [comment]
Fix, anything else to do ?

**Comment by dstftw** [comment]
Nothing conceptually changed.

**Comment by Urgau** [comment]
I disagree, I looked at the other extractors and I did the same thing they did.
Do you want a `try_get` ? What do you want ?

**Comment by dstftw** [comment]
Again: any new optional code addition **must not** affect existing extraction scenario. If webpage downloading fails it must not break extraction. If extraction of any additional metadata fails it must not break extraction. Extraction of additional metadata must happen after the main extraction scenario.

**Pull request closed without merging**